corehq/util/datadog/lockmeter.py

```diff
@@ -3,6 +3,8 @@
 
 import time
 
+from ddtrace import tracer
```

Comment:
We're fine assuming this library is installed on all environments, right? Just making sure we've used this pattern elsewhere without any negative consequences.

Reply:
It's right at the top of the form processor module, so I assume the answer is yes.

https://github.com/dimagi/commcare-hq/blob/0e7ca9e222dc56e712b7c8260ed81677b1eaf379/corehq/form_processor/submission_post.py#L8